---
title: Symposia, workshops, & discussion sessions
description: >
  Organized sessions at TDWG 2023 include symposia, workshops, posters, discussions, demos, contributed oral presentations, & keynotes.
background:
  img: https://images.unsplash.com/reserve/d1Ntvq9mSVmV0RcnWN1Y_23rd%20Studios%20Photography%20Boulder%20Colorado.jpg
  by: Paul Talbot
  href: https://unsplash.com/photos/pQDBGxtiDEo
toc: true
---

_Last updated 31 May 2023_

<!-- ## Keynote speakers -->

## Contributed oral presentations

This category reflects a broad range of author-submitted topics that are not part an organized session. Multiple sessions are anticipated.

## Posters

All those submitting poster abstracts will be required to create a virtual poster and will have the option to upload a short video to accompany it. At this time, the availability of space for printed posters has yet to be determined.

## Symposia

{:id="sym01"}
### SYM01 Biosecurity data: Why does it matter and how can access be facilitated?

Session type
: Symposium (lightning talks followed by panel discussion; open to outside abstract submissions)

Organizers
: Erin Roger, Atlas of Living Australia, Sydney, NSW, Australia, <erin.roger@csiro.au>; Andrew Turley, Atlas of Living Australia, Brisbane, Queensland, Australia, <andrew.turley@csiro.au> 

Evaluating and understanding the distribution and spread of pests, weeds and diseases through collation of data and big data analytics is a critical need for biosecurity systems globally. Biosecurity efforts aim to prevent, eradicate or mitigate impacts from pests and diseases, but prevention and effective responses to threats are only possible if reliable data is available for surveillance, risk analysis, response modelling and informed decision making. Efforts are currently hampered by limited findability, accessibility, and interoperability of biosecurity data. In addition, data is often fragmented across multiple platforms as well as socially, among program leaders, participants, researchers, and managers. Issues with ontology and data standardisation are also challenges, and inconsistent taxonomy further compounds these issues. This session will explore global case studies that examine the practices and pathways that facilitate better access to biosecurity data. Participants will engage with best practice, lessons learned, and concrete tools that they can use in their programs. We aim to strengthen both the cooperation and transfer of experiences between industry, research, community and government, and present solutions to current biosecurity data challenges.

{:id="sym02"}
### SYM02 Building bridges: Mapping from, to and between TDWG standards 

Session type
: Symposium: lightning talks (5-10 min) depending on number of submissions; open to outside abstract submissions; [related workshop](#wksh-02-building-bridges-mapping-from-to-and-between-tdwg-standards) to follow

Organizers
: David Fichtmueller, Botanic Garden and Botanical Museum Berlin, Freie Universität Berlin, Berlin, Germany, <d.fichtmueller@bgbm.org>; Holly Little, National Museum of Natural History, Smithsonian Institution, Washington D.C., United States of America, <LittleH@si.edu>; Mareike Petersen, Museum für Naturkunde Berlin, Leibniz Institute for Evolution and Biodiversity Science, Berlin, Germany, <Mareike.Petersen@mfn.berlin>
 
TDWG offers a large variety of standards, covering many aspects of geo- and biodiversity sciences. Each standard has its own purpose and area of focus, but there are always conceptual overlaps between them. Being able to systematically express and document those overlaps in the form of mappings is important for their interoperability, e.g., when merging data from different sources given in different standards. Mappings also help people who are familiar with one standard to better understand other standards with which they are less accustomed. 

However additional efforts are needed to better integrate mappings into the TDWG process and to provide a more comprehensive view of the relationships between the standards. Many standards lack mappings despite a conceptual overlap with others or the mappings that do exist are incomplete, inconsistently specified, and hard to find. 

This symposium offers an entry point into the discussion of mappings. We will review the mappings that already exist for various TDWG standards, the different technical approaches to specify mappings, and share a high-level view of the TDWG standards landscape to identify missing mappings.

This session will be followed by a [workshop](#wksh-02-building-bridges-mapping-from-to-and-between-tdwg-standards) for further hands-on exploration and contribution to standards mapping efforts.

{:id="sym03"}
### SYM03 Providing descriptive data based on the FAIR principles

Session type
: Symposium (open to outside abstract submissions)

Organizers
: Adeline Kerner, Center for Research on Palaeontology - Paris (UMR 7207: CNRS-MNHN-Sorbonne University), Paris, Ile-de-France, France, <kerner@mnhn.fr>

The identification and description of species and other taxa are the fundamental basis for every biodiversity study. Descriptive data takes a number of very different forms, more or less complete, structured or not, standardized or heterogenous. The importance of a standard for this data has been crucial since the beginnings of TDWG. To address this, following the Description Language for Taxonomy ([DELTA](https://www.tdwg.org/standards/delta/); ratified in 1986), the [SDD](https://www.tdwg.org/standards/sdd/) (Structured Descriptive Data) was ratified in 2005. Since then, a rapid growth and increasing quantities of computerized descriptive data is observed. In parallel, the techniques and languages to collect and manage them have evolved considerably with, for instance, AI (Artificial Intelligence), ontologies, text mining, and annotation. New platforms are developed, with some still using and adapting the SDD, and others not. It is critical in a FAIR (Findable, Accessible, Interoperable, and Reusable) data context to maintain models for sharing descriptive data that are structured enough to become knowledge, allowing comparison, identification, data-mining and application of analysis tools. In this session, we will review the existing models and discuss how to evolve the standard around the structuration of taxonomic descriptions.

{:id="sym04"}
### SYM04 The CARE principles: Learning and growing as a data management community

Session type
: Symposium (open to outside abstract submissions)

Organizers
: Abby Benson, U.S. Geological Survey, Denver, Colorado, United States of America, <albenson@usgs.gov>; Stephen Formel, U.S. Geological Survey, New Orleans, Louisiana, United States of America, <sformel@usgs.gov>; Peggy Newman, Atlas of Living Australia, Melbourne, Victoria, Australia, <peggy.newman@csiro.au>

The CARE Principles (Collective Benefit, Authority to Control, Responsibility, and Ethics) for Indigenous data governance  provide an important framework for ensuring that Indigenous and marginalized communities have control over the data they collect, data where they are the subject, or data collected in their territories (Carroll et al. 2020). Implementing the CARE Principles, alongside the FAIR Principles (Findable, Accessible, Interoperable, and Reusable), can minimize harm and maximize benefits to Indigenous and marginalized communities by accommodating past power and decision-making imbalances in the use of data. The data management community has an ethical responsibility to protect the intellectual rights of Indigenous knowledge holders. The entire biodiversity community would benefit from giving apt recognition to past and present power differentials between colonising and Indigenous cultures in data and knowledge exchange. This session will present the CARE principles and give examples of how they have been implemented around the world. We seek to increase our understanding of how CARE is being implemented, create a space for discussing approaches for applying CARE to biodiversity data, and bolster our ability to support Indigenous data sovereignty.

{:id="sym05"}
### SYM05 AI Contributions to biodiversity data & data standardisation: Opportunities and challenges 

Session type
: Symposium (open to outside abstract submissions)

Organizers
: Rukaya Johaadien, GBIF Norway, University of Oslo, Oslo, Norway, <rukayaj@gmail.com>; Michal Torma, GBIF Norway, University of Oslo, Oslo, Norway, <michal.torma@nhm.uio.no>; Kit Lewers, BioFrontiers Institute, and CUMNH (University of Colorado Boulder), Boulder, Colorado, United States of America, <kristen.lewers@colorado.edu>

Biodiversity data standardisation is an ongoing process with many challenges. Recently launched artificial intelligence (AI)-driven Large Language Models (LLMs) such as [ChatGPT-4](https://openai.com/product/gpt-4) have a new and unprecedented potential to contribute to the development/improvement of biodiversity data standards and the processes of standardisation. This is because, as LLMs are scaled, they reach critical levels where new abilities emerge suddenly and unpredictably. It is this emergent nature of LLMs, which has been fuelling the exponential jumps in their usage that we have seen over the past few months. By leveraging the new power of these AI language models, we can analyse and understand large amounts of biodiversity data, identify patterns and relationships, and derive insights that can improve our data standards and standardisation processes.

This session aims to explore the ways in which AI technology can contribute to the development and improvement of biodiversity data standards and the processes of standardisation.

Topics of interest include:

- The future role of AI in curating and developing data standards
- AI-based biodiversity data extraction and normalisation techniques
- AI-assisted biodiversity data mapping and standards alignment
- AI-driven metadata management and ontology development for biodiversity data
- Ethical and legal considerations in AI-assisted biodiversity data standardisation

{:id="sym06"}
### SYM06 Standards and automation for machine observations

Session type
: Symposium (open to outside abstract submissions)

Organizers
: Peter Desmet, Research Institute for Nature and Forest (INBO), Brussels, Brussels-Capital Region, Belgium, <peter.desmet@inbo.be>; Abby Benson, U.S. Geological Survey, Denver, Colorado, United States of America, <albenson@usgs.gov>; Peggy Newman, Atlas of Living Australia, Melbourne, Victoria, Australia, <peggy.newman@csiro.au>; Holger Dettki, Swedish Biodiversity Data Infrastructure (SBDI), Uppsala, Uppland, Sweden, <holger.dettki@slu.se>; Jonathan Pye, Ocean Tracking Network (OTN), Halifax, Nova Scotia, Canada, <jonpye@gmail.com>

Machine observations can be defined as species occurrence records that are captured by an automated trigger or process. Such methods include camera traps, marine imaging, GPS loggers, acoustic telemetry, radar and other techniques that record spatial and non-spatial sensor data. They offer enticing opportunities for recording biodiversity rapidly, continuously, in detail and across broad spatial scales. The large data volumes and processing needs of these methods however require automated workflows (e.g., species identification) and appropriate data exchange formats. In this session, we want to learn about recent advancements in this domain and invite presentations on:

1. Automated workflows from sensor capture to storage, from raw to processed species occurrence data
2. Challenges and opportunities in sharing and publishing machine observations (e.g., data exchange formats, sensitive data)

{:id="sym07"}
### SYM07 Ten Years of the Living Atlases

Session type
: Symposium (closed to outside abstract submissions)

Organizers
: Peggy Newman, Atlas of Living Australia, Melbourne, Victoria, Australia, <Peggy.Newman@csiro.au>; Javier Molina, Atlas of Living Australia, Melbourne, Victoria, Australia, <Javier.Molina@csiro.au>; Vicente J. Ruiz Jurado, GBIF.es and LA Community, Bimenes, Asturias, Spain, <vjrj@gbif.es>; Anne-Sophie Archambeau,  IRD/PatriNat/GBIF France, Paris, Ile de France, France, <archambeau@gbif.fr>; Francisco Pando, Real Jardín Botánico - CSIC, Madrid, Comunidad de Madrid, Spain, <pando@gbif.es>; Sophia Ratcliffe, NBN Atlas, United Kingdom, <s.ratcliffe@nbn.org.uk> 

The [Living Atlases](https://living-atlases.gbif.org/) community has grown and developed since 2013 by building national biodiversity data portals using open source software based on the Atlas of Living Australia ([ALA](https://www.ala.org.au/)). As the software evolves in collaboration with the Global Biodiversity Informatics Facility ([GBIF](https://www.gbif.org/)), and each platform seeks to address its own regional challenges, the Living Atlas portals have diverged into several permutations of architectural components. The pandemic has limited opportunity for workshops and in-person contact the community previously enjoyed.

For the first time after a long hiatus, we would like to invite participants in the Living Atlas community to meet face-to-face in Australia and share their experiences in a series of lightning talks and a roundtable discussion. Lightning talks should outline the current data portal architecture, works in progress and priorities for each platform. The ensuing moderated discussion will be an opportunity to raise issues and opportunities related to the software, and also discuss future innovation and collaboration pathways.

{:id="sym08"}
### SYM08 Exchange and reuse of environmental DNA and metabarcoding data

Session type
: Symposium (open to outside abstract submissions)

Organizers
: Tobias Guldberg Frøslev, Global Biodiversity Information Facility, Copenhagen, Sealand, Denmark, <tfroeslev@gbif.org>; Saara Suominen, Ocean Biodiversity Information System, Amsterdam, Netherlands, <s.suominen@unesco.org>

Metabarcoding of DNA is becoming a standard tool for detecting species in bulk samples or environmental samples such as soil, water, and air. In the name of reproducibility, raw sequence data (fastq files), code and metadata files are most often shared and deposited in dedicated repositories within the International Nucleotide Sequence Database Collaboration, GitHub, and in generalist repositories such as Dryad and Zenodo. However, seen from a biodiversity perspective, raw sequences have narrower applications compared to the interpreted DNA metabarcoding data. Most often such interpretations come in the shape of a table with sequence abundances of Amplicon Sequence Variants or Operational Taxonomic Units across samples – a so-called ASV/OTU-table – and some associated files, e.g., spatiotemporal and other sample metadata, and taxonomic inferences of sequences. In most cases these are at best part of the supplementary materials, and unfortunately rarely shared in a standardised way that facilitates reuse and integration of DNA data with other data types and data layers, and across studies. In this session we will present and invite presentations on improving, promoting, and assisting FAIR publication and standardisation of biodiversity data (OTU-table-like data) for easier and broader reuse, and integration with other biodiversity data and data layers.

{:id="sym09"}
### SYM09 Globally linking taxonomies: Name IDs and name matching 

Session type
: Symposium (open to outside abstract submissions)

Organizers
: Walter G. Berendsohn, Freie Universität Berlin, Botanic Garden and Botanical Museum Berlin, Berlin, Germany, <w.berendsohn@bo.berlin>; Markus Döring, GBIF Secretariat, Copenhagen, Denmark <mdoering@gbif.org>; Andreas Müller, Freie Universität Berlin, Botanic Garden and Botanical Museum Berlin, Berlin, Germany, <a.mueller@bo.berlin>; Anton Güntsch, Freie Universität Berlin, Botanic Garden and Botanical Museum Berlin, Berlin, Germany, <a.guentsch@bo.berlin>

Numerous local, regional and global information systems are using taxon names in fields ranging from biology, ecology and conservation to government provisions and medical applications. We know that scientific names of organisms do not provide a “unique identifier” for a taxon, because the taxon’s circumscription represents a scientific hypothesis and thus is subject to change. Nevertheless, names provide the glue that allows for information gathering and access across systems. Although scientific names are governed by strict codes of nomenclature, numerous variants of names exist. The taxonomic community is increasingly aware of this fact and several initiatives exist to sort out the latter issue. Globally unique identifiers for names and their variants play a central role allowing machines to support that task, and to conserve the results of the data cleaning actions. Name matching services are a tool to disperse GUIDs (Globally Unique Identifiers) to local users. One of the tasks of the EU [TETTRIs](https://tettris.eu/) project is to document and enhance these processes in order to establish functional workflows for the mapping of local taxonomies. The symposium will invite presentations and lightning talks to publicise existing services and persistent GUID systems with a special focus on long-term sustainability of the services.

{:id="sym10"}
### SYM10 Global biodata resources: A crucial infrastructure underpinning biodiversity research

Session type
: Symposium (open to outside abstract submissions)

Organizers
: Chuck Cook, Global Biodata Coalition, Strasbourg, Alsace, France, <ccook@globalbiodata.org>; Kathryn Hall, University of Queensland, Brisbane, Queensland, Australia, <Kathryn.Hall@csiro.au>; Jeff Christiansen, Australian BioCommons, Brisbane, Queensland, Australia, <jeff@biocommons.org.au>

Life science data resources are numerous, distributed and variously interconnected (or not), forming a singular, and arguably the largest, infrastructure for global biological research. These resources are critical for guaranteeing reproducibility and integrity for life sciences research, including biodiversity studies. Sustainably funding this disseminated infrastructure is a key challenge: the Global Biodata Coalition ([GBC](https://globalbiodata.org/)) is working with the funders who support many of these resources to ensure long-term funding for existing infrastructure, while also channelling support to underpin future growth in data volumes and new technologies.

In this session GBC will present an overview of its work to characterise the worldwide biodata infrastructure ([Global Core Biodata Resources](https://globalbiodata.org/scientific-activities/global-core-biodata-resources/) and an inventory of biodata resources) and will invite managers of data resources and aggregators to demonstrate the context of the entire infrastructure and to explore the scope and scale of connections and dependencies with other resources; the funding sources for the resources; and the impacts arising from the funding uncertainty associated with the underlying resources.

The goals of the session will be to raise awareness of the criticality of globally-connected infrastructure, its dependence on distributed resources, and the relative fragility of an infrastructure that is generally taken for granted by its users.

{:id="sym11"}
### SYM11 Combining biodiversity and environmental data for addressing scientific and societal questions

Session type
: Symposium (open to outside abstract submissions)

Organizers
: Francisco Pando, GBIF Spain / CSIC, Madrid, Spain, <pando@gbif.es>; William Ulate, Missouri Botanical Garden, St. Louis, Missouri, United States of America, <william_ulate_r@yahoo.com>

Most scientific questions as well and those related to conservation, policy or other societal challenges connected with the environment, cannot be approached on the basis of biodiversity data alone, but require other data types such as climate, soil, and land use. In this symposium we aim to bring together projects and use cases that, by combining biodiversity data with other types, produce visualizations, analyses or models.

We expect presentations to highlight methodological aspects such as data standards, or techniques to make different data sources work together to render coherent results. Projects focused on portals or Virtual Research Environments may fit especially well in this symposium.

We expect to gather a set of presentations illustrating varied approaches with different aims, from pure scientific scenarios to more applied ones (e.g., conservation, planning, climate change adaptation), from all-in-one comprehensive portals to specific solutions to make heterogeneous data compatible.

{:id="sym12"}
### SYM12 Improving the quality of taxonomic data publishing and list building

Session type
: Symposium (open to outside abstract submissions)

Organizers
: Olaf Bánki, Catalogue of Life, Amsterdam, Noord Holland, Netherlands, <olaf.banki@sp2000.org>; Thomas Orrell, ITIS/Smithsonian Institution, Washington DC, United States of America, <Orrellt@si.edu>; Donald Hobern, Species 2000, Canberra, Australian Capital Territory, Australia, <donald.hobern@sp2000.org>

Major improvements have been made in listing the world’s known species. The Catalogue of Life ([COL](https://www.catalogueoflife.org/)) Checklist now contains over 2,1 million species. The COL data infrastructure and services were recently rebuilt in collaboration with [GBIF](https://gbif.org/). This resulted in [ChecklistBank](https://www.checklistbank.org/), an open global repository for publishing and sharing taxonomic and nomenclatural datasets (>45,000 datasets, mostly mediated through [Plazi](https://plazi.org/)).
Despite these global efforts, COL is not perfect. There are data gaps (incompleteness), weak spots in several taxonomic groups (outdated data), data types that are challenging to integrate (e.g., paleodata) and a considerable time lag in being up-to-date with published literature.
Every day, taxonomists continue to publish new scientific names and refine our understanding of the world’s species. What is the role of taxonomic editing tools, such as those offered by [WoRMS](https://www.marinespecies.org/), [ITIS](https://www.itis.gov/), [TaxonWorks](https://taxonworks.org/), [WFO](http://www.worldfloraonline.org/), [EDIT](https://cybertaxonomy.org/) in managing and publishing data? Do we have commonly agreed taxonomic standards in place? To guarantee the highest confidence in data, we are in need of a transparent and consistent data quality assurance and data quality control mechanism from the creation of taxonomic data, to the data publishing, to the delivery into taxonomic data products and the global list of known species.

{:id="sym13"}
### SYM13 Publishing: Extending the biodiversity knowledge graph

Session type
: Symposium (closed to outside abstract submissions)

Organizers
: Donat Agosti, Plazi, Bern, BE, Switzerland, <agosti@plazi.org>; Laurence Bénichout, CETAF / Muséum national d'Histoire naturelle, Paris, Île de France, France, <laurence.benichou@mnhn.fr>; Lyubomir Penev, Pensoft Ltd., Sofia, Sofia, Bulgaria, <lyubo.penev@gmail.com>

A goal of natural history institutions is contributing to the understanding of biodiversity and disseminating this knowledge by becoming scientific publishers. Through this, they joined the growing field of scholarly publishing established in 1665 by the Journal des sçavans to publish work from the sciences. One of the characteristics of scientific publishing is to cite previous works and thus linking existing knowledge with new discoveries. To provide access to the growing corpus of publications, libraries began to index individual works. With the advent of the digital age, full text search and text and data mining provide access to the content using machines analysing large corpora of works. The development of the semantic web enabled creating a knowledge graph. A typical example is the growth of knowledge related to a species, including synonyms, cited specimens, figures or DNA sequences. During this symposium the state and recent developments of biodiversity publishing, linking and providing increased access to data by [Pensoft](https://pensoft.net/), [CETAF](https://cetaf.org/)-publishing group, [MNHN Paris](https://www.mnhn.fr/en), [Plazi](https://plazi.org/) and the [Swiss Institute of Bioinformatics](https://www.sib.swiss/), including over 50 journals in a semantic enhanced format, and supported by a series of projects ([BiCIKL](https://bicikl-project.eu/), [eBioDiv](https://ebiodiv.org/), [Arcadia fund](https://www.arcadiafund.org.uk/), [MétoSTeM](https://www.ouvrirlascience.fr/metostem-2/)) will be discussed.

{:id="sym14"}
### SYM14 Biodiversity informatics in Asia: Innovations & challenges

Session type
: Symposium (open to outside abstract submissions)

Organizers
: Vijay Barve, Natural History Museum of Los Angeles County, Los Angeles, California, USA, <vijay.barve@gmail.com>; Arjan Basu Roy, Nature Mates Nature Club, Kolkata, West Bengal, India, <naturemates@gmail.com>; Daphne Hoh, Taiwan Biodiversity Information Facility, Taipei, Taiwan, <daphnehohzhiwei@gmail.com>

The Asian continent is one of the most biodiverse regions in the world, boasting a diverse range of ecosystems and a multitude of endemic species. Biodiversity informatics plays a crucial role in conserving and understanding this richness. In this context, a growing community of practitioners in Asia is continuously developing new tools, technologies, and data-sharing platforms to support the study and conservation of biodiversity.

This symposium provides a platform to bring together active players in the field from across the continent to share their experiences and perspectives. Participants will have the opportunity to learn about interesting projects from across Asia, including novel approaches to data management, innovative tools, and new models for sharing and integrating biodiversity data. Additionally, the symposium will facilitate discussions on the challenges and opportunities of biodiversity informatics in Asia, such as the need for improved data quality and accessibility, the importance of engaging local communities in data mobilization efforts, and the potential of emerging technologies in biodiversity research.

{:id="sym15"}
### SYM15 Taxonomic backbone and catalogs: Solutions, workflows, and challenges

Session type
: Symposium (open to outside abstract submissions)

Organizers
: Vijay Barve, Natural History Museum of Los Angeles County, Los Angeles, California, USA, <vijay.barve@gmail.com> (soliciting additional organizers)

Taxonomy is the foundation of biodiversity research and informatics. However, managing taxonomic data can be challenging due to the complexities specific to each taxonomic group and management of names with different codes of nomenclature.

Several major initiatives are working towards developing and improving taxonomic backbones and catalogs, which are essential for facilitating access to and management of biodiversity data. The symposium aims to showcase various solutions, workflows, and challenges associated with developing and maintaining taxonomic backbones and catalogs for different groups of organisms.

The symposium will bring together researchers, curators, and practitioners from across the globe to share their experiences, approaches, and best practices related to taxonomic work. The presentations will highlight recent developments, innovative workflows, and future directions in developing and maintaining taxonomic backbones and catalogs.

The symposium will also provide an opportunity for participants to discuss the challenges associated with taxonomic research and explore strategies for overcoming them.

{:id="sym16"}
### SYM16 The role of collections management systems in the worldwide movement to roundtrip data enhancements

Session type
: Symposium (open to outside abstract submissions)

Organizers
: James Macklin, Agriculture and Agri-Food Canada, Ottawa, Ontario, Canada, <james.macklin@agr.gc.ca>; David Shorthouse, Agriculture and Agri-Food Canada, Ottawa, Ontario, Canada, <david.shorthouse@agr.gc.ca>; Falko Glöckler, Museum für Naturkunde Berlin, Berlin, Germany, <falko.gloeckler@mfn.berlin> 

There is renewed interest in a circular flow of annotations among and between isolated sources of biodiversity data, lately called data “round-tripping”. The structure and package for these annotations has assumed various new forms such as Frictionless Data, JATS XML, or nanopublications, but the fundamental principles and challenges remain unchanged.  Someone (or something) (1) has a valuable enhancement to make elsewhere from the source where original data or information are generated or transcribed; (2) wishes to broadcast these statements to the originator and to others who may benefit; and (3) expects persistence, discoverability, and attribution for their contributions alongside the source. Digital Extended Specimens must provide an answer to these socio-technical desires. This likewise puts pressure on collections management systems to quickly respond with functional integrations or risk being disconnected from the accumulation of valuable annotations. Besides the very obvious need for globally unique, persistent, resolvable identifiers for shared objects and concepts, we likewise require a universally consistent way to package and shuttle annotations that are sufficiently informative for a data curator to confidently make a decision. We look to the Web Annotation Working Group, Filtered Push, AnnoSys, Hypothes.is and their champions for guidance. Here, we propose a survey on what annotation frameworks and standards are presently in use in collections management systems for data objects, shared concepts, and the links among them, especially those that reach beyond the walls of a local, isolated system. We encourage contributors to reveal strengths, shortcomings, and challenges with the aims of reinvigorating the [Annotations Interest Group](https://www.tdwg.org/community/annotations/) and providing leadership through actionable recommendations. 

{:id="sym17"}
### SYM17 Describing biodiversity in underrepresented languages using crowdsourcing

Session type
: Symposium (open to outside abstract submissions)

Organizers
: Mohammed Kamal-Deen Fuseini, Gurene Wikimedia Community, Tamale, Northern, Ghana, <fuseinikamaldeen@gmail.com>; Abah Agnes Ajumah, Igbo Wikimedia User Group, Abuja, Federal Capital Territory, Nigeria, <ajben12@gmail.com>

Biodiversity, the variety of life on Earth, is crucial for conservation, but many languages, especially those spoken by indigenous communities, are underrepresented in scientific literature. This session explores the potential of crowdsourcing as a solution to collect biodiversity information in underrepresented languages.

Goals

- Highlight the challenge of underrepresented languages in documenting biodiversity
- Discuss the benefits of crowdsourcing for biodiversity data collection
- Share successful examples and best practices of crowdsourcing initiatives for biodiversity documentation

We hereby call on all researchers, conservationists, and language enthusiasts! Join our session to explore how crowdsourcing can revolutionize biodiversity documentation in underrepresented languages. Share your insights, experiences, and success stories in utilizing crowdsourcing platforms to collect biodiversity data from local communities. We invite abstract submissions that highlight innovative approaches, validation mechanisms, and user-friendly features in crowdsourcing initiatives. Let's work together to bridge the language gap, tap into the wealth of local knowledge, and enhance our understanding of global biodiversity for effective conservation efforts. Don't miss this opportunity to be part of a cutting-edge discussion and contribute to the advancement of biodiversity research and conservation!

## Demonstration

{:id="demo01"}
### DEMO01 The BiCIKL toolbox to link biodiversity data

Session type
: Demonstration (closed to outside abstract submissions)

Organizers
: Quentin Groom, Meise Botanic Garden, Meise, Vlaams-Brabant, Belgium, <quentin.groom@plantentuinmeise.be>;  Olaf Banki, Species 2000 / Catalogue of Life, Amsterdam, , Netherlands, <olaf.banki@sp2000.org>; Patrick Ruch, HES-SO, Carouge, Geneva, Switzerland, <patrick.ruch@hesge.ch>; Wouter Addink, Naturalis Biodiversity Center, Leiden, Netherlands, <wouter.addink@naturalis.nl>; Lyubomir Penev, Pensoft Publishers, Sofia, Bulgaria, <l.penev@pensoft.net>; Donat Agosti, Plazi,  Bern, Switzerland, <agosti@plazi.org>

We will demonstrate tools and infrastructure from the [BiCIKL](https://bicikl-project.eu/) project to find and establish links between entities. These include:
- Infrastructure to create and manage machine actionable persistent identifiers (PIDs) for digital specimens. This allows the creation and annotation of digital specimens as new actionable objects on the internet, which provide a surrogate for physical specimens.
- [OpenBiodiv](https://openbiodiv.net/) is a linked open data (LOD) knowledge graph extracted from the biodiversity literature. OpenBiodiv can discover hidden links, e.g., between authors, taxa, sequences, material citations, publications and others using general search, SPARQL (query language), user applications and API.
- [TreatmentBank](https://plazi.org/treatmentbank/) liberates data from within publications, and converts, enhances, links, stores, and disseminates it as findable, accessible, interoperable and reusable (FAIR) data. The API has been enhanced allowing better data linkages through identifiers, matching services, and avenues for annotation. 
- The [SIBiLS](https://www.expasy.org/resources/sibils) custom search services provide access to biomedical literature, including PubMed Central ([PMC](https://www.ncbi.nlm.nih.gov/pmc/)) and [MEDLINE](https://www.nlm.nih.gov/medline/index.html). It also provides access to half a million taxonomic treatments curated by [Plazi](https://plazi.org/). BICIKL has helped aggregate more publications, enabling the creation of an all-inclusive "One Health" library, the BiodiversityPMC.
- [ChecklistBank](https://www.checklistbank.org/) is an open data repository with a focus on taxonomy and nomenclature. It has tools for name matching, dataset comparison, and for linking taxonomic names from literature.

## Panel Discussion

{:id="pd01"}
### PD01 Data quality: Lessons and directions

Session type
: Panel discussion (closed to outside abstract submissions)

Organizers
: Lee Belbin, Blatant Fabrications Pty Ltd, Hobart, Tasmania, Australia, <leebelbin@gmail.com>; Arthur Chapman, Australian Biodiversity Information Services, Ballan, Victoria, Australia, <biodiv_2@achapman.org> 

The [Tests and Assertions Task Group](https://www.tdwg.org/community/bdq/tg-2/) has been working on a standard for testing fitness for  use for over five years. To get to this point, we acknowledge and value the high level of skill and perseverance of the core team: Arthur Chapman, John Wieczorek, Paul Morris, and Lee Belbin. The project has been epic in many ways.

We have come a long way, primarily in erecting a set of 99 'tests' and associated specifications. We are most comfortable with the specifications as they provide a concise yet comprehensive framework for understanding the tests, implementing them and adding new domain specific tests as required. At the time of writing, minor refinements are evolving as a result of implementing the outstanding tests and running our test data against the implementation.

We hope that the discussion at this session will familiarise participants with what we have done. We will also use the session to answer questions, gain feedback, provide advice on implementations and seek support for any outstanding issues, either known at the time, or arising.

## Workshops

{:id="wksh01"}
### WKSH01 Using AWK to check for errors, inconsistencies and anomalies between fields in Darwin Core records

Session Type
: Workshop (in person only; closed to outside abstract submissions)

Organizer
: Robert Mesibov, West Ulverstone, Australia, <robert.mesibov@gmail.com> 

Quality tests for Darwin Core tables are mostly checks within individual fields:

- are all data items valid and appropriate for that field?
- are all data items correctly and consistently formatted?
- are any data items missing?

However, a "correct" entry in one field might disagree with a "correct" entry in another field within the same record. These errors can only be detected with between-field tests. A few simple between-field tests have been implemented by biodiversity data aggregators, for example:

- is minimumElevationInMeters less than maximumEIM?
- do decimalLatitude/decimalLongitude agree with country?

but many more between-field tests are possible.

This 2-hour, hands-on workshop is aimed at data publishers and data curators who need to check for between-field problems when preparing or cleaning Darwin Core data. Participants should be familiar with the BASH shell and will need to bring a laptop running BASH (on Windows, Mac or Linux), with GNU AWK installed. A few functions will be used but no shell scripts, and sample data will be provided. The session includes a brief introduction to the AWK text-processing language, which can easily, quickly and flexibly deal with Darwin Core tables with up to several million records.

Questions? Email the presenter <robert.mesibov@gmail.com>

More information:

- [Field disagreements in Darwin Core](https://discourse.gbif.org/t/field-disagreements-in-darwin-core/3370)
- [A Data Cleaner's Cookbook](https://www.datafix.com.au/cookbook/)
- [What is AWK?](https://opensource.com/resources/what-awk)
- [learnbyexample: GNU AWK](https://learnbyexample.github.io/learn_gnuawk/)
- [Wikipedia: AWK](https://en.wikipedia.org/wiki/AWK)

{:id="wksh02"}
### WKSH02 Building bridges: Mapping from, to and between TDWG standards

Session type
: Hands-on workshop (in-person and virtual participation; closed to outside abstract submissions)

Organizers
: Mareike Petersen, Museum für Naturkunde Berlin, Leibniz Institute for Evolution and Biodiversity Science, Berlin, Germany, <Mareike.Petersen@mfn.berlin>; Holly Little, National Museum of Natural History, Smithsonian Institution, Washington D.C., Washington D.C., United States of America, <LittleH@si.edu>; David Fichtmueller, Botanic Garden and Botanical Museum Berlin, Freie Universität Berlin, Berlin, Germany, <d.fichtmueller@bgbm.org>

To gain critical background for participating in this workshop, please attend the [associated symposium](#sym02) that will introduce this topic, looking at existing mappings, technical approaches, and identifying missing mappings.

This workshop offers a hands-on approach to explore options for contributing to mappings, to provide input on the expected process, and to gather the needs and requirements from the community. Participation will be possible both in person and virtually.

{:id="wksh03"}
### WKSH03 Handling restricted access data in an open data landscape

Session type
: Workshop (closed to outside abstract submissions)

Organizers
: Cameron Slatyer, Atlas of Living Australia, Canberra, ACT, Australia, <cam.slatyer@csiro.au>; Andrew Rodrigues, Global Biodiversity Information Facility, Copenhagen, Hovedstaden, Denmark, <arodrigues@gbif.org>; Tania Laity, tania.laity@csiro.au, Canberra, ACT, Australia, <tania.laity@csiro.au>

The growth of biodiversity data aggregators and citizen science projects over the past two decades has led to an exponential expansion in the application of big data to conservation assessment and biodiversity research. Subsequently, expectations that biodiversity data should be made available have risen, as has the recognition of the challenges of both releasing and not releasing sensitive species data.
 
As biodiversity data has been democratised, the motivations of individuals, researchers, and organisations in restricting access to certain types of data have remained constant. Data producers and custodians may have a more conservative view of access than data consumers. Factors to be taken into consideration include type and level of threat, vulnerability, type of information, and public availability. Additionally, there is a growing recognition of the need to enable Indigenous peoples and local communities to assert data sovereignty over traditional knowledge and biodiversity data gathered by, about or within areas managed by them. 
 
The focus of this workshop is to develop sensitive data standards that respect both FAIR (Findable, Accessible, Interoperable, and Reusable) and CARE (Collective Benefit, Authority to Control, Responsibility, and Ethics) principles and enable practical data sharing, which can be incorporated into Darwin Core.

{:id="wksh04"}
### WKSH04 Working session to develop genomics-specific extensions with the Darwin Core event model

Session type
: Workshop (closed to outside abstract submissions)

Organizers
: Kathryn Hall, Australian Reference Genome Atlas (ARGA) - Atlas of Living Australia, Australian BioCommons and Bioplatforms Australia, with the Australian Research Data Commons, Canberra, ACT, Australia, <kathryn.hall@csiro.au>

Fundamental to the work of the Australian Reference Genome Atlas ([ARGA](https://arga.org.au/)) project has been the attempt to standardise the way in which we ingest and handle genomics data. Not all data are equal: single locus data is very different from whole genome annotations. As our work has progressed, we have identified a number of fields that are common across all of the records we are indexing, and useful subsets that pertain to specific data-types, many of which are amenable to the development of additional vocabularies. We would like to invite the TDWG community to join with us for an extended working session to work out the details of some of the extensions that we would like to propose adding to the genomics extensions blocks. We would welcome any feedback on the work we have done and also welcome the contributions of others who are developing standards and controlled vocabularies in this space. By the end of the session, we plan to submit some new standards and vocabularies for further consideration and refinement by the community.

{:id="unconf"}
### TDWG 2023 Unconference

Session type
: Unconference (no abstracts required)

Organizers
: Abigail Benson, U.S. Geological Survey, Lakewood, CO, USA, <albenson@usgs.gov>; Dimitri Brosens, Flemish Research Institute for Nature and Forest, Brussels, Belgium <dimitri.brosens@inbo.be>; Shelley James, Western Australia Dept. of Biodiversity Conservation and Attractions, Kensington, WA, Australia <shelley.james@dbca.wa.gov.au>; Deborah Paul, Illinois Natural History Survey, Prairie Research Institute, University of Illinois, Champaign, IL, USA <dlpaul@illinois.edu>

TDWG attendees will have the opportunity to take control of the conference programme for 60 minutes of unbridled open participant-driven discussion, demonstration and debate. Do you have a topic you want to discuss but haven’t seen it in any of the sessions? Did one of the sessions spark an idea for you and you want to explore it further? Have recent events spurred subjects you want to talk about with other TDWG folks? Do you have an application idea or analysis you want to explore with others? If so, then the unconference is for you and we want you to propose your ideas. What topics need deeper discussion and exploration? The unconference is a place to collaborate, discover, explore and take a deep dive into emerging topics of interest.
